examples/classdiagram/src/model-source.ts

```diff
@@ -145,6 +145,7 @@ export class ClassDiagramModelSource extends LocalModelSource {
         const node0 = {
             id: 'node0',
             type: 'node:class',
+            name: 'Foo',
```

Comment:
This leads to redundant information in the model: the name is present in the `name` attribute and in the text of the `label:heading` type label. This might be a problem when one wants to rename an element. It could be solved in this case by replacing `name` with a getter property that retrieves the value of the heading label.